Sources/WebSocketKit/Concurrency/WebSocket+Concurrency.swift

```diff
@@ -60,19 +64,41 @@ extension WebSocket {
         }
     }
 
+    @preconcurrency public func onPong(_ callback: @Sendable @escaping (WebSocket, ByteBuffer) async -> ()) {
```

Comment:
Actually since this is a new API we can force users to use it with safe code - I'll take care of this
```suggestion
    public func onPong(_ callback: @Sendable @escaping (WebSocket, ByteBuffer) async -> ()) {
```